docs/doxygen/gh_pages.py

```diff
@@ -61,7 +61,7 @@ def main():
                      ['/{}'.format(correction) for correction in defunct_files]
     corrections_out = ['"{}'.format(correction) for correction in corrected] + \
                       ['/{}'.format(correction) for correction in corrected]
-    full_files = [os.path.join(path, fle) for path, _, files in os.walk(directory) for fle in files]
+    full_files = [os.path.join(path, file) for path, _, files in os.walk(directory) for file in files]
```

Comment:
hope this isn't a problem

Reply:
This can be a problem as `file` is a python builtin.  Should be safe within the list comprehension, so I think it is ok.